enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -2103,6 +2103,10 @@ func (p *PebbleCache) Writer(ctx context.Context, r *rspb.ResourceName) (interfa
 	if err != nil {
 		return nil, err
 	}
+	sizeBytes := r.GetDigest().GetSizeBytes()
```

Comment:
For AC entries the digest size does not equal the content size. Could we skip updating the metric for chunks and only update it for non-chunk entries?